Apollo.xcodeproj/project.pbxproj

```diff
@@ -3851,6 +3859,16 @@
 			package = 9B7BDAF423FDEE2600ACD198 /* XCRemoteSwiftPackageReference "SQLite.swift" */;
 			productName = SQLite;
 		};
+		DE3A2815268BCE6700A1BDC8 /* Starscream */ = {
+			isa = XCSwiftPackageProductDependency;
+			package = DE8C84F2268BBF8000C54D02 /* XCRemoteSwiftPackageReference "Starscream" */;
+			productName = Starscream;
+		};
+		DE8C84F3268BBF8000C54D02 /* Starscream */ = {
+			isa = XCSwiftPackageProductDependency;
+			package = DE8C84F2268BBF8000C54D02 /* XCRemoteSwiftPackageReference "Starscream" */;
+			productName = Starscream;
+		};
```

Comment:
also not clear on why this has been added like three times

Reply:
Not clear on that either... XCProject files still have some weird bugs with their SPM integrations sometimes. I'm not too worried about it though. Not sure if there is anything we can do to fix that...